stable/atlantis/templates/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "atlantis.fullname" . -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ template "atlantis.name" . }}
+    chart: {{ template "atlantis.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+{{- with .Values.ingress.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.ingress.tls }}
+  tls:
+{{ toYaml .Values.ingress.tls | indent 4 }}
+{{- end }}
+  rules:
+    - host: {{ .Values.ingress.host }}
+      http:
+        paths:
+          - path: {{ .Values.ingress.path }}
+            backend:
+              serviceName: {{ $fullName }}
```

Comment:
Should match the name of the service `{{ template "atlantis.fullname" . }}`

Reply:
I believe it already does, per line 2 of this same file.  Please mark resolved or correct me if I've got it wrong.